### ISSUE TITLE
elasticjob-lite-ui Can't run in Windows environment

### ISSUE THREAD

**Issue by baiyijun** [apache/shardingsphere-elasticjob-ui#129]
This software cannot run in the windows environment, because some files in the tar package do not end with .jar, I think it is caused by the maven plug-in, but I cannot solve it
like this:
![image](https://user-images.githubusercontent.com/11145358/130178125-d007a3bd-3a64-46de-98fc-0a42b33a37f1.png)

**Comment by sunkai-cai** [comment]
Could you provide more information about it?

**Comment by baiyijun** [comment]
When I run / bin / start.bat, the system prompts that "org.apache.shardingsphere.elasticjob.lite.ui.bootstrap" cannot be found
![image](https://user-images.githubusercontent.com/11145358/130205045-ff570b06-b930-4802-8805-3d04153076a6.png)
Then I looked at the files in the "lib" folder,I found that the end of many files is not ". Jar", it seems to be truncated, so I compared it and found that as long as the file name cannot exceed 45 letters.
like this
![image](https://user-images.githubusercontent.com/11145358/130206156-b866bd1b-3b3a-4e1c-9088-6947c84cc383.png)
Next, I downloaded the source code and compiled it. It looks normal, but after using "shardingsphere-elasticjob-Lite-UI-bin-distribution.xml" to generate the ". tar" file, the jar file name is truncated
